resources/js/Pages/Vault/Journal/Post/Edit.vue

```diff
@@ -76,6 +85,45 @@ const debouncedWatch = debounce(() => {
   update();
 }, 500);
 
+const onSuccess = (file) => {
+  form.uuid = file.uuid;
+  form.name = file.name;
+  form.original_url = file.originalUrl;
+  form.cdn_url = file.cdnUrl;
+  form.mime_type = file.mimeType;
+  form.size = file.size;
+
+  upload();
+};
+
+const upload = () => {
+  saveInProgress.value = true;
+
+  axios
+    .put(props.data.url.upload_photo, form)
+    .then((response) => {
+      saveInProgress.value = false;
+      localPhotos.value.push(response.data.data);
+    })
+    .catch((error) => {
+      form.errors = error.response.data;
+    });
+};
+
+const destroyPhoto = (file) => {
+  if (confirm('Are you sure? The photo will be deleted immediately.')) {
```

Comment:
We should avoid using the `confirm` handle, it's not really beautiful ...
I've started using the `ConfirmationModal` instead, do you want to use it here?

Reply:
Good idea, i'll do that.

Reply:
done